tools/cardano-sl-tools.cabal

```diff
@@ -17,6 +17,12 @@ Flag with-web
 
   description: Build with web part.
 
+Flag with-post-mortem-tool
+  default:     False
+  manual:      True
+
+  description: Build with web part.
```

Comment:
@adinapoli-iohk we have `for-installer` flag in `cardano-sl-tools`, if this flag is enabled, only tools necessary for installer will be built (e. g. `cardano-launcher`). This flag is enabled in appveyor and macos builds and disabled in linux builds. Linux build times are not an issue, so we want to build everything there.
https://github.com/input-output-hk/cardano-sl/blob/0d28e6133bd6349f5236bcebab39ea6bfc4c2b7e/scripts/ci/travis.sh#L31
So, tl;dr: `for-installer` flag should be used instead of a standalone flag for `post-mortem-tool`.